rust/xaynet-client/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "xaynet-client"
+version = "0.9.0"
+authors = ["Xayn Engineering <engineering@xaynet.dev>"]
+edition = "2018"
+description = "The Xayn Network project is building a privacy layer for machine learning so that AI projects can meet compliance such as GDPR and CCPA. The approach relies on Federated Learning as enabling technology that allows production AI applications to be fully privacy compliant."
+license-file = "../../LICENSE"
+keywords = ["federated-learning", "fl", "ai", "machine-learning"]
+repository = "https://github.com/xaynetwork/xaynet/"
+homepage = "https://xaynet.dev/"
+
+[dependencies]
+tokio = { version = "0.2.21" }
+derive_more = { version = "0.99.7", default-features = false, features = ["from"] }
+serde = { version = "1.0.111", features = ["derive"] }
+bytes = "0.5.4"
+sodiumoxide = "0.2.5"
+bincode = "1.2.1"
+thiserror = "1.0.19"
+tracing = "0.1.16"
+async-trait = "0.1.35"
+xaynet-core = { path = "../xaynet-core" }
+xaynet-server = { path = "../xaynet-server" }
```

Comment:
This can actually be a dev-dependency because it is only needed for the in-memory client, which is only compiled for the tests:

```
❯ rg server
Cargo.toml
23:xaynet-server = { path = "../xaynet-server" }

src/api/in_memory.rs
10:use xaynet_server::services::{FetchError, Fetcher, PetMessageError, PetMessageHandler};
```

Atm we don't even use it in the tests though, so I'm not even sure it's worth keeping but that may be a debate for another PR.

Fwiw I found this using [`cargo-udeps`](https://github.com/est31/cargo-udeps), which is a pretty useful tool.

Reply:
good catch 👍
I fixed it 
Yeah I agree. I think the idea was to keep it for integration tests later.